e2e/ansible/inventory/group_vars/all.yml

```diff
@@ -24,15 +24,6 @@ weave_version_list:
   - 1.9.4
   - 2.0.4 
 
-#######################################
-#     OpenEBS deployment type         #
-#######################################
-#Option to specify OpenEBS deployment type
-# It can be either kubectl or helm based installation
-
-openebs_deploy: helm
```

Comment:
Let us not remove this. This flag is being used to decide openebs deployment mode for CI